wherehows-web/app/utils/api/datasets/compliance.ts

```diff
@@ -132,6 +141,32 @@ const readDatasetComplianceSuggestionByUrn = async (urn: string): Promise<ICompl
   return complianceSuggestion;
 };
 
+/**
+ * Reads the export policy for a dataset by urn
+ * @param urn
+ * @return {Promise<IDatasetExportPolicy>}
+ */
+const readDatasetExportPolicyByUrn = async (urn: string): Promise<IDatasetExportPolicy> => {
+  let exportPolicy: IDatasetExportPolicy = <IDatasetExportPolicy>{};
+
+  try {
+    ({ exportPolicy = <IDatasetExportPolicy>{} } = await getJSON<Pick<IDatasetExportPolicyResponse, 'exportPolicy'>>({
+      url: datasetExportPolicyByUrn(urn)
+    }));
```

Comment:
There is nothing wrong with this, but it took me a while to undestanding. Wouldnt it easier to read like this?:

```
 exportPolicy = (await getJSON<Pick<IDatasetExportPolicyResponse, 'exportPolicy'>>({
      url: datasetExportPolicyByUrn(urn)
    }))).exportPolicy;
```

Reply:
not quite the same, missing the default value for exportPolicy